src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4510,66 +4510,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
-  /api/repository/v1/datasets/{id}/snapshotBuilder/settings:
```

Comment:
Would it also be in scope for this ticket to remove the getSnapshotSnapshotBuilderSettings endpoint?

Reply:
Do we want to remove that endpoint? I thought we would want to keep that - at least until we add an option to the retrieveSnapshot endpoint to return a snapshot with its settings. I could file a ticket for that change if we want to do that.

Reply:
I think we discussed leaving `getSnapshotSnapshotBuilderSettings` as a separate endpoint/making snapshot builder settings have their own separate REST path. @pshapiro4broad might remember, I think he had thoughts here?

Reply:
We have a new endpoint to return the snapshot builder settings: https://data.terra.bio/swagger-ui.html#/snapshots/getSnapshotSnapshotBuilderSettings

So, I don't think we need the dataset based endpoint.

Reply:
I'm sorry, I was confused - I thought there was an equivalent getSnapshotBuilderSettings endpoint for datasets still hanging around.